### PR TITLE
Add Starlight 0.28 blog post

### PR DESCRIPTION
**Pull request by delucis**
This PR adds a blog post aboout Starlight v0.28

## Browser Test Checklist

I have tested this PR on at least three of the following browsers:

- [x] Chrome / Chromium
- [x] Firefox
- [x] Android Firefox
- [x] Safari
- [ ] iOS Safari

### CODE REVIEW

File: src/content/blog/starlight-028.mdx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+}),
+```
+
+As a bonus, with this syntax you get link checking for free — no more accidental 404s! Thanks to [Lorenzo](https://github.com/lorenzolewis) for adding this feature.
+
+## Sparkling fresh docs
+
+And we've been working on our own docs, too!
+We overhauled the documentation for Starlight’s built-in components.
+Each component now has a dedicated page including common usage examples and a full prop reference.
```
Comment (aFuzzyBear) [inline src/content/blog/starlight-028.mdx:117]:
```suggestion
We've been working hard on our own docs, too!
Overhauling all the documentation for Starlight’s built-in components.
Now, each component has their own dedicated page including common usage examples and a fully documented `prop` references.
```

File: src/content/blog/starlight-028.mdx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+Visit the new [“Using components” guide](https://starlight.astro.build/components/using-components/) to explore the available components.
+
+Of course, these wouldn’t be Astro docs without translations.
+Shout out to `@dreyfus92`, `@jsparkdev`, `@dragomano`, `@liruifengv`, `@zhoushengdao`, `@Nin3lee`, `@casungo`, `@at-the-vr`, and `@thomasbnt` for keeping our Spanish, Korean, Russian, Simplified Chinese, Italian, Hindi, and French docs updated!
```
Comment (aFuzzyBear) [inline src/content/blog/starlight-028.mdx:122]: can we link to github profiles?

File: src/content/blog/starlight-028.mdx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+## A Markdoc preset for the adventurous
+
+Starlight users interested in Markdoc can use the new `@astrojs/starlight-markdoc` package to quickly configure all of Starlight’s built-in components for use in Markdoc files.
+See our [Markdoc set-up guide](https://starlight.astro.build/guides/authoring-content/#markdoc) for more details.
+
+## Try Starlight today!
+
+We’ve been excited to see companies like [Cloudflare](https://developers.cloudflare.com/), [Crowdin](https://support.crowdin.com/), [StackBlitz](https://tutorialkit.dev/), [Sentry](https://spotlightjs.com/), [Font Awesome](https://docs.fontawesome.com/), and [SST](https://sst.dev/docs/) embrace Starlight when building docs. Join them and open-source projects like [FreeCodeCamp](https://contribute.freecodecamp.org/), [RealWorld Apps](https://www.realworld.how/), [Biome](https://biomejs.dev/), [Jimp](https://jimp-dev.github.io/jimp/), [Shepherd](https://docs.shepherdpro.com/), [Bloc](https://bloclibrary.dev/), and [Knip](https://knip.dev/) — to name just a few! — and get started today.
```
Comment (aFuzzyBear) [inline src/content/blog/starlight-028.mdx:131]: 💚 I loved reading this, wow 🤯

File: src/content/blog/starlight-028.mdx [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+```sh
+npm create astro@latest -- --template starlight
+```
+
+## Previously…
+
+import RelatedPosts from './_components/RelatedPosts.astro';
+
+<RelatedPosts slugs={['starlight-turns-one', 'wp-engine']} />
```
Comment (aFuzzyBear) [inline src/content/blog/starlight-028.mdx:143]: ques que say?

**Review by aFuzzyBear — COMMENTED** [review]
Im blown away by the progress Starlight has had, this looks to be a truly awesome release

💚

File: src/content/blog/starlight-028.mdx [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+```sh
+npx @astrojs/upgrade
+```
+
+## Next-level localization
+
+Starlight has been multilingual from day one and v0.28 takes this to the next level.
+Starlight’s built-in localization system is now powered by the first-in-class [`i18next`](https://www.i18next.com/) library and available to use anywhere in your site.
+
+Use the new `Astro.locals.t()` method from any Astro component to render Starlight’s built-in UI labels or your own custom strings, including support for interpolation and pluralization:
```
[the remark this answers is not in the document]
Comment (delucis) [reply src/content/blog/starlight-028.mdx:33]: Ah yeah, the new docs weren’t out yet. Will add a link now!

**Review by HiDeoo — APPROVED** [review]
Amazing, such a lovely blog post :star2:

**Pull request merged**